api/v1beta1/webhook_suite_test.go

```
@@ -0,0 +1,415 @@
+	cancel()
+	By("tearing down the test environment")
+	err := testEnv.Stop()
+	Expect(err).NotTo(HaveOccurred())
+})
+
+var _ = Describe("Model Registry validating webhook", func() {
+	// TODO add tests for defaulting webhook and MR config validation
+
+	It("Should not allow creation of duplicate MR instance in cluster", func(ctx context.Context) {
+		Eventually(func() error {
+			config.SetRegistriesNamespace("") // run this test with no ns restrictions
```

golangci-lint (1.64.8)

Error return value of `config.SetRegistriesNamespace` is not checked

(errcheck)